public/app/plugins/datasource/prometheus/querybuilder/components/NestedQuery.tsx

```diff
@@ -68,14 +69,7 @@ export const NestedQuery = React.memo<Props>(({ nestedQuery, index, datasource,
   );
 });
 
-const operators = [
-  { label: '/', value: '/' },
-  { label: '*', value: '*' },
-  { label: '+', value: '+' },
-  { label: '==', value: '==' },
-  { label: '>', value: '>' },
-  { label: '<', value: '<' },
-];
+const operators = binaryScalarDefs.map((def) => ({ label: def.sign, value: def.sign }));
```

Comment:
Also used the defs here so it's in sync I assume all are also valid for nested queries